### PR TITLE
feat(perf): Add a new span operations widget

### PR DESCRIPTION
**Pull request by udameli**
This PR adds a new span ops widget that shows a stacked bar series of `p75(spans.db)`, `p75(spans.resource)`, `p75(spans.http)`, `p75(spans.ui)`, `p75(spans.browser)`. The new widget is hidden behind a server-side sampling flag as it uses processed data.

Example for frontend transactions that have >2 span ops
Screen Shot 2022-12-13 at 12 40 04 PM

Example for backend transactions that have 2 span ops
Screen Shot 2022-12-13 at 12 40 15 PM

Fixes PERF-1857

### CODE REVIEW

**Comment by udameli** [comment]
> Are db spans supposed to be on the top or bottom image
> 
> When I viewed this, it was sometimes on the top (like your screenshot) and sometimes on the bottom.

Yeah, it depends which span op came in first in the response. It's not meant to be deterministic except for the fact that we wanted lighter colors first (which is why I reversed the color palette)

File: static/app/views/performance/transactionSummary/utils.tsx [hunk 1]
```diff
@@ -20,8 +20,14 @@ export enum TransactionFilterOptions {
...
-export function generateTransactionSummaryRoute({orgSlug}: {orgSlug: string}): string {
-  return `/organizations/${orgSlug}/performance/summary/`;
+export function generateTransactionSummaryRoute({
+  orgSlug,
+  subPath,
+}: {
+  orgSlug: string;
+  subPath?: string;
+}): string {
+  return `/organizations/${orgSlug}/performance/summary/${subPath ? subPath + '/' : ''}`;
```
Comment (DominikB2014) [inline static/app/views/performance/transactionSummary/utils.tsx:30]: Nit, but I do think it looks cleaner even with the nested template strings
```suggestion
  return `/organizations/${orgSlug}/performance/summary/${subPath ? `${subPath}/` : ''}`;
```

**Review by DominikB2014 — APPROVED** [review]
LGTM and consistent to rest of the widgets. Some level testing would be good to add too!

File: static/app/views/performance/transactionSummary/utils.tsx (hunk 1 above)
Comment (udameli) [reply static/app/views/performance/transactionSummary/utils.tsx]: for sure! thanks!

**Comment by udameli** [comment]
> LGTM and consistent to rest of the widgets. Some level testing would be good to add too!

Added some basic render tests 👍 More granular new accordion component tests are coming!

**Pull request merged**